lib/ts/recipe/session/sessionAuth.tsx

```diff
@@ -161,8 +160,42 @@ const SessionAuth: React.FC<PropsWithChildren<SessionAuthProps>> = ({ children,
         }
     }, []);
 
+    const getRedirectPathFromInvalidClaimsOrRedirect = async (claims: ClaimValidationError[]) => {
+        if (!claims.length) {
+            return;
+        }
+        const globalValidators: SessionClaimValidator[] = getGlobalClaimValidators(props.overrideGlobalClaimValidators);
```

Comment:
Do we need to specify the type here?

Reply:
yes, web-js | website does not know about callbacks